### DISCUSSION TITLE
在我封装多层之后（期间有tsx）ant design vue的输入框会自动弹出建议

### DISCUSSION THREAD

**Discussion by adoin** [vueComponent/ant-design-vue#7753] [Q&A]
弹出的div有pseudo="-internal-input-suggested"的属性
这个要怎么修复啊

**Comment by adoin** [comment]
![bug](https://github.com/user-attachments/assets/4d6b345c-6520-401f-9b50-59c82de97519)

**Comment by adoin** [comment]
在输入框上 autocomplete="off" 能禁用
但是我怎么给AInput传这个值呢

**Comment by ryne6** [comment]
https://cn.vuejs.org/guide/components/attrs.html#fallthrough-attributes
透传下去可以吗

**Reply by adoin** [reply]
都不用透，给默认值就行。
我的问题其实就是封装多层之后，他的默认值为啥和我其他未封装的不一样，结果就是我把出问题的几个组件重新继承一下默认配置就行。
